packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -139,12 +141,14 @@ export const prepareAuctionBook = (baggage, zcf) => {
         priceBook,
         scaledBidBook,
 
-        assetsForSale: AmountMath.makeEmpty(collateralBrand),
         collateralSeat,
         curAuctionPrice: zeroRatio,
         currencySeat,
         lockedPriceForRound: zeroRatio,
         updatingOracleQuote: zeroRatio,
+        // undefined indicates no limit; empty indicates limit exhausted
+        /** @type {Amount<'nat'> | undefined} */
+        totalProceedsGoal: undefined,
```

Comment:
IIRC we agreed to have `null` explicit "no goal". Also the comment will appear on hover if in the jsdoc
```suggestion
        /** null indicates no limit; empty indicates limit exhausted
         *  @type {Amount<'nat'> | null} */
        totalProceedsGoal: null,
```